lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -250,8 +249,6 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
             }
         }
 
-        this.browserStorage.updateCacheEntries(validatedRequest.state, validatedRequest.nonce, validatedRequest.authority, validatedRequest.loginHint || "", validatedRequest.account || null);
```

Comment:
Changes in this file, like the RedirectClient.ts look unrelated to the intent of the PR, maybe you need to re-sync with other updates.

Reply:
updating cache entries is part of the "extensiblity" changes, those are now done in their respective interaction handlers